### PR TITLE
blas,lapack: enable code generation without $GOPATH

### PR DESCRIPTION
**Pull request by vladimir-ch**
Please take a look.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -10,3 +10,5 @@ require (
 	modernc.org/strutil v1.0.0 // indirect
 	modernc.org/xc v1.0.0
 )
+
+go 1.13
```
Comment (vladimir-ch) [inline go.mod]: I'm not sure what this implies and whether this should go in. The go tool did this by itself.

Comment (kortschak) [reply go.mod]: This is fine.

Comment (vladimir-ch) [reply go.mod]: Ok, I'll keep it.

**Comment by vladimir-ch** [comment]
Added a commit after `go generate`.

**Comment by vladimir-ch** [comment]
Included in #67 , closing.

**Pull request closed without merging**